ui/admin/app/templates/scopes/scope/aliases/alias/index.hbs

```diff
@@ -23,12 +23,76 @@
       </nav.link>
     </Rose::Nav::Tabs>
   </page.navigation>
-
+  {{#if (can 'delete model' @model)}}
```

Comment:
Do you need delete permissions if you just want to clear?

Reply:
uh... I thought we did, checking with BE

Reply:
discussed offline, we don't need del permissions to clear, we removed the modal and listed the actions now